azkaban-web-server/src/web/js/azkaban/view/flow-execute-dialog.js

```diff
@@ -300,6 +300,8 @@ azkaban.FlowExecuteDialogView = Backbone.View.extend({
         if (node.type == "flow" && statusNode.type == "flow") {
           this.assignInitialStatus(node, statusNode);
         }
+      } else {
```

Comment:
Does node.missing refer to some  job being left in running or queued state incorrectly? Could you please add some comment in this else block?